tests/Mono.Android-Tests/Java.Interop/JavaListTest.cs

```diff
@@ -15,6 +15,38 @@ namespace Java.InteropTests
 		[SetUp]
 		public void Setup () => list = new T ();
 
+		[Test]
+		public void Add ()
+		{
+			list.Add ("foo");
+			Assert.AreEqual ("foo", list [0]);
+
+			// Ensure duplicates are allowed.
+			list.Add ("foo");
+			Assert.AreEqual (2, list.Count);
+			Assert.AreEqual ("foo", list [1]);
+		}
+
+		[Test]
+		public void AddWithIndex ()
+		{
+			list.Add ("Apple");
+			list.Add ("Banana");
+			list.Add ("Cherry");
+
+			// Ensure index is respected.
+			list.Add (3, "Grape");
+			list.Add (2, "Blueberry");
+			list.Add (4, "Fig");
+
+			Assert.AreEqual ("Apple", list [0]);
```

Comment:
This new test is failing on CI:

```
Expected is <System.String>, actual is <Java.Lang.ICharSequenceInvoker>
Values differ at index [0]
Expected: 'A'
But was:  'C'
```

The `SubList()` test is also failing:

```
Expected is <System.String>, actual is <Java.Lang.ICharSequenceInvoker>
Values differ at index [0]
Expected: 'f'
But was:  'b'
```

Very odd. :-(